src/sequelize/migrations/20190805003447-create-user.js

```diff
@@ -32,6 +32,11 @@ export const up = (queryInterface, Sequelize) => queryInterface.createTable('Use
     type: Sequelize.STRING,
     defaultValue: 'user',
   },
+  status: {
```

Comment:
this should be a boolean field. Refer to: https://github.com/andela/coding-geeks-ah-backend/pull/14#discussion_r312726200

Reply:
Done